### DISCUSSION TITLE
Welcome to cyberdisc-bot Discussions!

### DISCUSSION THREAD

**Discussion by thebeanogamer** [CyberDiscovery/cyberdisc-bot#597] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jerbob** [comment]
![image](https://cdn.discordapp.com/emojis/750350748746907669.gif?v=1)

**Reply by 0xAda** [reply]
![68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3735303335303734383734363930373636392e6769663f763d31](https://user-images.githubusercontent.com/8531139/101531499-43765d00-398b-11eb-9750-41ad2caf5976.gif)

**Comment by 0xAda** [comment]
![68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3735303335303734383734363930373636392e6769663f763d31](https://user-images.githubusercontent.com/8531139/101531160-c519bb00-398a-11eb-9e4b-5f9f767008c0.gif)

**Reply by jerbob** [reply]
![image](https://cdn.discordapp.com/emojis/750350748746907669.gif?v=1)

**Reply by 0xAda** [reply]
![68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3735303335303734383734363930373636392e6769663f763d31](https://user-images.githubusercontent.com/8531139/101531382-16c24580-398b-11eb-9d5e-ff783f5d648e.gif)

**Reply by thebeanogamer** [reply]
![](https://user-images.githubusercontent.com/8531139/101531382-16c24580-398b-11eb-9d5e-ff783f5d648e.gif)

**Reply by 0xAda** [reply]
![68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3735303335303734383734363930373636392e6769663f763d31](https://user-images.githubusercontent.com/8531139/101531499-43765d00-398b-11eb-9750-41ad2caf5976.gif)

**Reply by ConnorMcF** [reply]
![](https://user-images.githubusercontent.com/8531139/101531499-43765d00-398b-11eb-9750-41ad2caf5976.gif)

**Reply by 0xAda** [reply]
![68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3735303335303734383734363930373636392e6769663f763d31](https://user-images.githubusercontent.com/8531139/101531499-43765d00-398b-11eb-9750-41ad2caf5976.gif)

**Comment by ghost** [comment]
Github Discussions™️, I say we delete the server and migrate to GitHub Discussions™️. I will rewrite the cdbot to match the switch.

**Reply by thebeanogamer** [reply]
My God you're hilarious Jack. Truly the peak of comedy. I'm glad this shirt is tight, or I think my sides would have split.

**Reply by ghost** [reply]
![](https://user-images.githubusercontent.com/8531139/101531382-16c24580-398b-11eb-9d5e-ff783f5d648e.gif)
